core/templates/pages/admin-page/roles-tab/admin-roles-tab.component.html

```diff
@@ -183,7 +183,7 @@
                   class="btn btn-success protractor-test-contribution-rights-form-submit-button"
                   [disabled]="!formData.viewContributionReviewers.isValid()"
                   value="view role">
-            View Role
+            View Contributors
```

Comment:
Just making sure if this change was intentional

Reply:
yup this is intentional

Reply:
Can you please leave an explanation for this change? [This will help reviewers to quickly approve the changes!]

Reply:
![Screenshot from 2021-06-28 16-01-57](https://user-images.githubusercontent.com/54100702/123623048-a4cc2380-d82a-11eb-98c8-b8a2f7e8b727.png)

Currently, we have two categories of roles.
One is actual roles (admin, collection editor, moderator...)
Another one is contributor rights.
We are actually viewing contributors/ or contributor rights, In order to sync with other two buttons add rights and remove rights I've made this change.